main.js

```diff
@@ -1,173 +1,188 @@
+                reject(response.errorMessages[0]);
+                return;
+            }
+            resolve(response);
+        };
+
+        // Handle network errors
+        req.onerror = () => {
+            reject(Error("Network Error"));
+        }
+        req.onreadystatechange = () => {
+            if (req.readyState == 4 && req.status == 401) {
+                reject("You must be logged in to JIRA to see this project.");
+            }
+        }
+
+        // Make the request
+        req.send();
+    });
 }
 
-
-
-function loadOptions(){
-  chrome.storage.sync.get({
-    project: 'Sunshine',
-    user: 'nyx.linden'
-  }, function(items) {
-    document.getElementById('project').value = items.project;
-    document.getElementById('user').value = items.user;
-  });
+const loadOptions = () => {
+    chrome.storage.sync.get({
+        project: 'Sunshine',
+        user: 'nyx.linden'
+     }, (items) => {
+        document.getElementById('project').value = items.project;
+        document.getElementById('user').value = items.user;
+    });
 }
-function buildJQL(callback) {
-  var callbackBase = "https://jira.secondlife.com/rest/api/2/search?jql=";
-  var project = document.getElementById("project").value;
-  var status = document.getElementById("statusSelect").value;
-  var inStatusFor = document.getElementById("daysPast").value
-  var fullCallbackUrl = callbackBase;
-  fullCallbackUrl += `project=${project}+and+status=${status}+and+status+changed+to+${status}+before+-${inStatusFor}d&fields=id,status,key,assignee,summary&maxresults=100`;
-  callback(fullCallbackUrl);
+
+const buildJQL = (callback) => {
+    const callbackBase = "https://jira.secondlife.com/rest/api/2/search?jql=";
+    const project = document.getElementById("project").value;
+    const status = document.getElementById("statusSelect").value;
+    const inStatusFor = document.getElementById("daysPast").value
+    let fullCallbackUrl = callbackBase;
+    fullCallbackUrl +=
+        `project=${project}+and+status=${status}+and+status+changed+to+${status}+before+-${inStatusFor}d&fields=id,status,key,assignee,summary&maxresults=100`;
+    callback(fullCallbackUrl);
 }
-function createHTMLElementResult(response){
 
-// 
-// Create HTML output to display the search results.
-// results.json in the "json_results" folder contains a sample of the API response
-// hint: you may run the application as well if you fix the bug. 
-// 
+const createHTMLElementResult = (response) => {
+    document.getElementById('query-result').hidden = false;
+    let issues = response.issues.map((issue) =>{
+        return `<b>Summary: ${issue.fields.summary}</b><ul><li>Id: ${issue.id}</li><li>Status: ${issue.fields.status.name}</li><li>Link: <a href=${issue.self}>${issue.self}</a></li></ul>`
+    })
+    return `<div id ='query-issue-result'>
+      <h4>Issues: ${response.total}</h4>
+      <div>
+        ${issues.join('')}
+      </div>
+    </div>`;
```

Comment:
How did you decide on the display format to use?  This is very different than the activity query output (although they are not displaying the same type of data).